linode/manager
# M3-5451: Changes to Cluster Details for HA Clusters and Kubernetes Dashboard

## Description

This PR adds functionality for HA clusters to manager. These additions are:

- [x] Upgrade to HA button
- [x] HA cluster chip

Also included is the new feature for all clusters to reach an external K8s dashboard:
- [x] External link button to K8s dashboard
The external link ti K8s dashboard does not render if the API returns an error or no URL.

## How to test

**What are the steps to reproduce the issue or verify the changes?**

You must include `REACT_APP_LKE_HIGH_AVAILABILITY_PRICE` in your env to see new features.

**How do I run relevant unit tests?**
